okhttp/src/test/java/okhttp3/DuplexTest.java

```diff
@@ -505,6 +510,64 @@
     mockDuplexResponseBody.awaitSuccess();
   }
 
+  /**
+   * OkHttp currently doesn't implement failing the request body stream independently of failing the
+   * corresponding response body stream. This is necessary if we want servers to be able to stop
+   * inbound data and send an early 400 before the request body completes.
+   *
+   * This test sends a slow request that is canceled by the server. It expects the response to still
+   * be readable after the request stream is canceled.
+   */
+  @Ignore
```

Comment:
Is this shortlived?  If not, rather than ignoring, is it worth flagging with a TODO but stating the test in terms of how the failure is currently unfortunately observed?

Reply:
It’s hopefully shortlived. Need to do some implementation refactoring to make it work.